test/socket/test_tcp.rb

```diff
@@ -140,4 +140,168 @@ def test_accept_multithread
       server_threads.each(&:join)
     end
   end
+
+  def test_ai_addrconfig
```

Comment:
I wrote this test, but it's so complicated and environment dependent that I wonder if it's worth having at all - maybe I should delete this?

Reply:
Thank you for merging this!

I see you committed the skip in https://github.com/ruby/ruby/commit/abf192eb16ea845ac11743a32bd2f3e2d234488b. I agree this makes sense, even without the skip this test wouldn't actually do anything on CI anyway - `unshare(2)`  won't work without root privileges and presumably we don't run the tests as root in Github Actions or rubyci.org. So better to just skip and be clear about it.